opm/material/fluidmatrixinteractions/EclMaterialLawManager.hpp

```diff
@@ -242,6 +242,54 @@ class EclMaterialLawManager
         MaterialLaw::updateHysteresis(*threePhaseParams, fluidState);
     }
 
+    template <class Evaluation>
```

Comment:
I'm pretty sure that none of the parameter classes needs to deal with AD. (hence only `Scalar` but no `Evaluation` parameters should be required.)